test/files/run/splain.check

```diff
@@ -0,0 +1,62 @@
+newSource1.scala:13: error: implicit error;
+!I e: II
+ImplicitChain.g invalid because
+!I impPar3: I1
+⋮
+――ImplicitChain.i1 invalid because
+  !I impPar7: I3
+  implicitly[II]
+            ^
+newSource1.scala:6: error: type mismatch;
+  L|R
```

Comment:
Compared to the default message
```
 found   : FoundReq.L
 required: FoundReq.R
```
this doesn't contain the prefix. I'm not sure exactly what the logic is in the default messages, what prefixes are kept. Maybe worth taking a look?

Reply:
there is a lot of disambiguation logic and configuration options for prefixes in the plugin, though some of that is still outstanding for backport (see conversation with @SethTisue above) or has been removed (it was requested to cut down the number of options somewhere).

So this is already on the agenda, we can add options back later if we deem them essential!

Reply:
👍